### ISSUE TITLE
[1.0.3] Applied Energistics 2 crash due to Forge version. 

### ISSUE THREAD

**Issue by AzureDoom** [AllTheMods/ATM-8#320]
The version of Applied Energistics 2 being shipped with the client and server files is crashing the server due Forge version being lower. 
```
[23.11 01:09:22] [Server] -- MOD ae--
[23.11 01:09:22] [Server] Details:
[23.11 01:09:22] [Server] 	Mod File /mods/appliedenergistics2-forge-12.8.6.jar
[23.11 01:09:22] [Server] 	Failure message Mod ae2 requires forge 43.1.55 or above, and below 44.0.0
[23.11 01:09:22] [Server] 		Currently, forge is 43.1.52
[23.11 01:09:22] [Server] 	Mod Version 12.8.6
```

**Comment by MAXimized490** [comment]
Can confirm. This can be resolved by using the 43.1.55 forge installer instead of the one supplied by the pack.

**Comment by Mitchell5200** [comment]
Version 1.0.3 has been re-uploaded with the correct forge version now.
Forge is now 43.1.57
Update, Closing.
